README.md

```diff
@@ -1,2 +1,3 @@
 # 7-HelloWorld
-This is to fulfill assignment A4
```

Comment:
You shouldn't remove this line from the readme. This line comes from the main branch and in theory has already been "reviewed" and "approved", in reality it was just the initial creation Nick made. Regardless, we generally do not make changes to the original file unless it is the point of the code changes. In this case, the assignment was to add to the readme on the main branch and it is what we have all done thus far. Your changes may cause a merge conflict, which in this case is easy to fix, but for best practices, I would ask that you re-add this line and commit and push the changes.